### PR TITLE
Patch card to support workloads/tagging/sids 🚤✅🥰

### PR DESCRIPTION
**Pull request by AllenBW**
fixes https://issues.redhat.com/browse/ADVISOR-1405

reverted app.js global filter handling to suite apps in place

#### looks LIIIIKEEEEEEEE this

Screen Shot 2020-10-15 at 12 29 16 PM
Screen Shot 2020-10-15 at 12 28 19 PM

### CODE REVIEW

File: src/App.js
Comment (karelhala) [inline src/App.js:38]: Can you revert this change? As the new API of `mapGlobalFilter` allows passing 3rd parameter to preformat the data for you? https://github.com/RedHatInsights/insights-chrome/blob/f0eb20df8671392fc0d3ef9aad6eb40b64659fb9/README.md#prefered-way-of-consuming-data-in-structured-format

Comment (AllenBW) [reply src/App.js:38]: this change breaks the dashboard workload selector (neither inventory or advisor cards work with workloads) furthermore, at least in prod-beta, SIDs dont exist yet n this change gobbles up tags, shows em as SIDS

Comment (AllenBW) [reply src/App.js:38]: also, regarding the link sent, is this work on prod beta? not seeing     // workloads = { SAP: { isSelected: true } } as the default value returned when using the mapGlobalFilter

Comment (karelhala) [reply src/App.js:38]: I forgot the API is still not merged to each environment... Yeah we can roll out with the older version of mapping global filter and I'll open new PR to enable the newer way once the API lands.

File: src/App.js
Comment (karelhala) [inline src/App.js:42]: Since we don't have the new way of mapping filter you might want to make this into an array
```JS
dispatch(setSIDs(Object.entries(data['SAP ID (SID)']).filter(([, { isSelected }]) => isSelected).map(([ key ]) => key)));
```

File: src/App.js
Comment (karelhala) [reply src/App.js:38]: The formatted API is just on ci/qa-beta and is blocked by API unfortunately.

**Review by karelhala — DISMISSED** [review]

File: src/SmartComponents/Advisor/Advisor.js
Comment (LI0131) [inline src/SmartComponents/Advisor/Advisor.js:59]: Would it be worthwhile to move `sapFilter` into the common utils file?

**Review by LI0131 — COMMENTED** [review]
Looking good @AllenBW 😄

File: src/SmartComponents/Advisor/Advisor.js
Comment (AllenBW) [reply src/SmartComponents/Advisor/Advisor.js:59]: hmmmm well every card that supports, will be using, and yah see we do the same thing in both patch and inv card... lemme give it a try n <3

**Review by karelhala — APPROVED** [review]

**Review by LI0131 — APPROVED** [review]

**Pull request merged**
